Find a functionally equivalent code:android.view.accessibility.AccessibilityEvent.setSealed:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Sets if this instance is sealed.
 *
 * @param sealed Whether is sealed.
 *
 * @hide
 */

@@ -1,9 +1,11 @@
 {
     super.setSealed(sealed);
-    List<AccessibilityRecord> records = mRecords;
-    final int recordCount = records.size();
-    for (int i = 0; i < recordCount; i++) {
-        AccessibilityRecord record = records.get(i);
-        record.setSealed(sealed);
+    final List<AccessibilityRecord> records = mRecords;
+    if (records != null) {
+        final int recordCount = records.size();
+        for (int i = 0; i < recordCount; i++) {
+            AccessibilityRecord record = records.get(i);
+            record.setSealed(sealed);
+        }
     }
 }

